includes/woopay/class-woopay-session.php

```diff
@@ -385,6 +385,10 @@ private static function get_checkout_data( $woopay_request ) {
 	 */
 	private static function get_user_email( $user ) {
 		if ( ! empty( $_POST['email'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			if ( is_array( $_POST['email'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+				 // phpcs:ignore WordPress.Security.NonceVerification, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
+				return sanitize_email( wp_unslash( WooPay_Utilities::decrypt_signed_data( $_POST['email'] ) ) ) ?? '';
```

Comment:
It looks like there's a chance `WooPay_Utilities::decrypt_signed_data( $_POST['email'] )` may return `null`. Can `wp_unslash( ... )` handle `null`?

Reply:
Yes, both `wp_unslash` and `sanitize_email` can handle `null`. In case the return of `decrypt_signed_data` is `null`, the final output is an empty string 👍